libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI+Notifications.h

```diff
@@ -75,6 +81,11 @@ NS_SWIFT_NAME(FetchNotificationsRequestBuilder)
 
 @end
 
+/**
+ * Use this interface to create a RestRequest object that calls the NotificationUpdate and NotificationsUpdate REST API.
```

Comment:
Are there actually NotificationUpdate/NotificationsUpdate endpoints? I think the updates are just the Notifications API PATCH  methods (as your links correctly imply).

Reply:
Ah whoops, I thought that's how you said we'd be referring to them. How about "Use this interface to create a PATCH RestRequest object that calls the Notifications REST API"?

Reply:
I did say that--my bad! 👎 What you suggest is perfect.